### PR TITLE
update wording

### PR DESCRIPTION
**Pull request by maryamsulemani97**
Update note wording

### CODE REVIEW

File: learn/advanced/updating.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ If Meilisearch launches successfully, use the get version endpoint, note your `p
 <CodeSamples id="updating_guide_check_version_new_authorization_header" />
 
 ::: note
-If you're using v0.24 or below, use the `X-MEILI-API-KEY: apiKey` authorization header:
+If you get a [`missing_authorization_header`](/reference/errors/error_codes.md#missing-authorization-header) error code, you might be using v0.24 or below. Change the authorization header to `X-MEILI-API-KEY`
```
Comment (maryamsulemani97) [inline learn/advanced/updating.md:24]:
```suggestion
If you get a [`missing_authorization_header`](/reference/errors/error_codes.md#missing-authorization-header) error code, you might be using v0.24 or below. Change the authorization header to `X-MEILI-API-KEY`:
```

File: learn/advanced/updating.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ If Meilisearch launches successfully, use the get version endpoint, note your `p
 <CodeSamples id="updating_guide_check_version_new_authorization_header" />
 
 ::: note
-If you're using v0.24 or below, use the `X-MEILI-API-KEY: apiKey` authorization header:
+If you get a [`missing_authorization_header`](/reference/errors/error_codes.md#missing-authorization-header) error code, you might be using v0.24 or below. Change the authorization header to `X-MEILI-API-KEY`:
```
Comment (maryamsulemani97) [inline learn/advanced/updating.md:24]:
```suggestion
If you get a [`missing_authorization_header`](/reference/errors/error_codes.md#missing-authorization-header) error code, you might be using v0.24 or below. Change the authorization header to `X-MEILI-API-KEY: apiKey`:
```

File: learn/advanced/updating.md [hunk 3]
```diff
@@ -21,9 +21,9 @@ If Meilisearch launches successfully, use the get version endpoint, note your `p
 <CodeSamples id="updating_guide_check_version_new_authorization_header" />
 
 ::: note
-If you're using v0.24 or below, use the `X-MEILI-API-KEY: apiKey` authorization header:
+If you get a [`missing_authorization_header`](/reference/errors/error_codes.md#missing-authorization-header) error code, you might be using v0.24 or below. Change the authorization header to `X-MEILI-API-KEY: apiKey`:
```
Comment (guimachiavelli) [inline learn/advanced/updating.md:24]: Probably nitpick, I just felt "return an error code" was more specific "get".

```suggestion
If Meilisearch returns a [`missing_authorization_header`](/reference/errors/error_codes.md#missing-authorization-header) error code, you might be using v0.24 or below. Change the authorization header to `X-MEILI-API-KEY: apiKey`:
```

**Review by guimachiavelli — DISMISSED** [review]
Minor nitpick—accept it you think it's a good suggestion, but PR approved either way 🌗

**Review by guimachiavelli — APPROVED** [review]
🦘

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**